.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ version: 2.1
 jobs:
   publish_release:
     docker:
-      - image: circleci/android:api-29
+      - image: circleci/openjdk:8-jdk
```

Comment:
Why the change to a more generic docker image?

Reply:
originally it was openjdk.  I ran into an issue with the installed gradle version not being compatible with the android plugin.  I tried using the android image as a solution, but have decided against it.  It didn't have a different installed gradle version.  I have switched to using the gradle wrapper as a fix.  If we start building apks or doing android tests we may need to change, but also need to consider iOS, so think that is part of a bigger solution.